velox/functions/prestosql/Bitwise.h

```diff
@@ -28,6 +30,29 @@ struct BitwiseAndFunction {
   }
 };
 
+template <typename T>
+struct BitCountFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE bool call(int64_t& result, TInput num, TInput bits) {
+    if (bits == kMaxBits) {
+      result = std::bitset<kMaxBits>(num).count();
+      return true;
+    }
+    if (bits <= 1 || bits > kMaxBits) {
+      return false;
+    }
+    // check if input "num" falls within the limits of max and min that
```

Comment:
Comments should be full sentences, e.g. start with a capital letter and end with a period.